### ISSUE TITLE
Multiple common snd and fightfx

### ISSUE THREAD

**Issue by RealFoobs** [ikemen-engine/Ikemen-GO#746]
**Is your feature request related to a problem? Please describe.**
fightfx.sff and fightfx.air made for the mugen engine aren't really forwards compatible since they do not include the necessary sprites and animations for Ikemen's native systems like stun. While the sprites themselves are provided in a very clean manner already, a complete newbie might not know how to export those to an existing sff file.

**Describe the solution you'd like**
Enable the engine to load multiple fightfx.sff, fightfx.air and common.snd, much in the same way it can already load multiple cns/zss common files. Make it so files like dizzy.zss can access any of these files.

**Additional context**
This feature would also open up the doors for more people to develop content like lifebars, input display tools or Mortal Kombat fatality systems making extensive use of explods, without those creating compatibility issuess among themselves.

**Comment by Lazin3ss** [comment]
The current idea we have for this feature is to add new settings to config file, where "commonSff" and "commonSnd" files could be declared. A "preffix" (much like F in fightfx) would also be able to be declared for them, so they could be referenced in character code. CommonAir could also be expanded with this feature in mind. For example:

In config file:
```
commonAir = {
    {"data/MortalKombatFx.air", "MK"}
}
commonSff = {
    {"data/MortalKombatFx.sff", "MK"}
}
commonSnd = {
    {"data/MortalKombatFx.snd", "MK"}
}
```

Character code (ZSS):
```coffeescript
explod{ID: 20; anim: MK40}
PlaySnd{value: MK2,0; channel: 1}
```

**Comment by potsmugen** [comment]
This would solve the issue of dizzy and guard break effects disappearing when people change the fightfx. Those could just ship with the engine as commonsff's and commonsnd's (D and G?)

**Comment by Lazin3ss** [comment]
This issue has been addressed by #817 and thus can be closed.
